src/main/java/eu/qped/framework/Translator.java

```diff
@@ -80,4 +81,13 @@ public void translateStyleBody(String pref, StyleFeedback feedback) {
 
         }
     }
+
+    public void translateDesignBody(String pref, DesignFeedback feedback) {
```

Comment:
Ich würde keinen Code mit Release mergen, der noch nicht bereit ist. Bitte entweder die Methode vervollständigen oder gar nicht in Release mergen.